### PR TITLE
Add a doc on website preview

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Added a document that explains how to access website previews through Netlify links 

**Related issue(s)**
See also #https://github.com/kyma-project/website/issues/285

### CODE REVIEW

**Comment by kazydek** [comment]
@derberg I talked to @magicmatatjahu and the preview for the `community` was off at that time. I tested it now (adding indents to this PR) and created a brand new PR and it works -> https://github.com/kyma-project/community/pull/403.

File: guidelines/content-guidelines/10-docs-preview.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: Documentation preview
+---
+
+Before you publish new documentation on the website, you can preview your changes to see if the formatting of the text is correct, images fit well, and links work as expected.
+
+That is possible thanks to the **preview feature** supported by [Netlify](https://www.netlify.com/). It attaches links to autogenerated previews of all website views to your pull requests (PRs).
+
+This feature is enabled on these Kyma repositories:
+
+- [`kyma`](https://github.com/kyma-project/kyma/tree/master/docs) for changes in the `/docs` folder that contains sources of the official Kyma documentation in the [**Docs**](https://kyma-project.io/docs/) view.
```
Comment (mmitoraj) [inline guidelines/content-guidelines/10-docs-preview.md]:
```suggestion
- [`kyma`](https://github.com/kyma-project/kyma/tree/master/docs) for changes in the `/docs` folder that contains sources of the official Kyma documentation rendered in the [**Docs**](https://kyma-project.io/docs/) view.
```

File: guidelines/content-guidelines/10-docs-preview.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- [`community`](https://github.com/kyma-project/community) for changes rendered in the [**Community**](https://kyma-project.io/community/) and [**Roadmap**](https://kyma-project.io/roadmap/) views.
+
+- [`website`](https://github.com/kyma-project/website) for changes rendered on the [landing page](https://kyma-project.io/) and in the [**Blog**](https://kyma-project.io/blog/) view.
+
+Previews are built for PRs containing changes made to any files within these repositories. The only exception is the `kyma` repository where Netlify only publishes notifications of successful builds on PRs that modify the content of the `/docs` folder.
```
Comment (mmitoraj) [inline guidelines/content-guidelines/10-docs-preview.md]:
```suggestion
Previews are built for PRs containing changes made to any file within these repositories. The only exception is the `kyma` repository where Netlify only publishes notifications of successful builds on PRs that modify the content of the `/docs` folder.
```

File: guidelines/content-guidelines/10-docs-preview.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- a direct link to log details for failed builds
+
+    ![Failed preview](./assets/failed-preview.png)
+
+The website preview deployment is also visible on the list of jobs that need to pass for the PR before it can be merged. Click on **Details** to go to the build logs.
+
+![Job details](./assets/job-details.png)
+
+>**NOTE:** For the `kyma` repository, Netlify only attaches notifications for successful preview builds. Thus, if you make any changes in the `/docs` folder and there is no website preview attached to your PR, access failed build information through the link under **Details** available next to the preview job.
```
Comment (mmitoraj) [inline guidelines/content-guidelines/10-docs-preview.md]: This information can be changed into **Troubleshooting** or you can leave it in the Note.

Comment (kazydek) [reply guidelines/content-guidelines/10-docs-preview.md]: Hmm, I think I will leave it as a note as it refers specifically to the `kyma` repo and in a way to the section on failed builds.

**Pull request merged**